src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -903,17 +906,31 @@ internal static Expression IsStrictMode(int version, Expression executionContext
                                    ExpressionCache.Constant(version));
         }
 
-        internal Expression UpdatePosition(Ast ast)
+        private int AddSequencePoint(IScriptExtent extent)
```

Comment:
Note: this method is added to make sure that one extent can only be added to the sequence list once.